### DISCUSSION TITLE
`@auth/sveltekit`: how to make working `adapter-node` build of the app?

### DISCUSSION THREAD

**Discussion by 0x416e746f6e** [nextauthjs/next-auth#10177] [Help] [answered]
hi!

I am new to svelte and `@auth/sveltekit`.  just made a small app that uses it with twitter provider:

https://github.com/0x416e746f6e/sveltekit-authjs-twitter/tree/auth-sveltekit-question

in dev mode all works as expected, however once I build it and run with `node build/index.js` (or with `node express.js`) the authentication flow "breaks" at the stage where the client sends `POST` to `/auth/signin/twitter` path and gets `403` for response.

is there a way to fix this? 

I tried enabling pre-rendering of all pages with `export const prerender = true;` in `+layout.server.ts` but that doesn't help (it breaks the build actually, and moreover, I am not sure that's really a good idea since paths under `/auth` are dynamic by nature).

**Comment by weepy** [comment]
I have this problem - it used to work - but after upgrading various packages it doesn't :(

**Reply by 0x416e746f6e** [reply, marked as answer]
I eventually figured it out, but don't remember now how exactly.

f.w.i.w., here's the repo:  https://github.com/flashbots/eth-faucet/tree/main/frontend/static
